### ISSUE TITLE
Pre built binarries missing (windows x86, node 12)

### ISSUE THREAD

**Issue by dansgeo** [nexe/nexe#784]
**What happened**:
Automated build was working last week. Rebuilt today with minor change and no change to nexe config and getting :
```
Nexe Build Failed NexeError: https://github.com/nexe/nexe/releases/download/v3.0.0/windows-x86-12.15.0 is not available, create it using the --build flag
```
I get a 404 if i manually hit that link and can't see any binaries for that tag in the github releases

**What you expected to happen**:
pre-built binaries downloaded

**How to reproduce it (as minimally and precisely as possible)**:
"nexe": "^3.3.2",
building on debian with latest node  12.x
targeting win32-x86

**Anything else we need to know?**:
Running in a docker container

**Environment**:
- Platform(OS/Version): Debian
- Host Node Version: 12.x (assume 12.15.0)
- Target Node Version: not sepcified
- Nexe version: 3.3.2
- Python Version: ?

**Comment by calebboyd** [comment]
A change was made changing the external releases tag that should've have been and its been fixed in 3.3.3

**Comment by kevstev01** [comment]
Seeing this problem again : can't find https://github.com/nexe/nexe/releases/download/v3.3.3/windows-x86-12.18.4 (or 12.18.3 too)
